eng/code-quality-reports/src/main/resources/checkstyle/checkstyle.xml

```diff
@@ -195,7 +195,17 @@ page at http://checkstyle.sourceforge.net/config.html -->
         <!-- See http://checkstyle.sf.net/config_whitespace.html -->
         <module name="EmptyForIteratorPad"/>
         <module name="MethodParamPad"/>
-        <module name="NoWhitespaceAfter"/>
+        <module name="NoWhitespaceAfter">
+            <property name="tokens" value="INC"/>
+            <property name="tokens" value="DEC"/>
+            <property name="tokens" value="UNARY_MINUS"/>
+            <property name="tokens" value="UNARY_PLUS"/>
+            <property name="tokens" value="BNOT"/>
+            <property name="tokens" value="LNOT"/>
+            <property name="tokens" value="DOT"/>
+            <property name="tokens" value="ARRAY_DECLARATOR"/>
+            <property name="tokens" value="INDEX_OP"/>
+        </module>
```

Comment:
I assumed the tokens property should be a single, comma-separated string, rather than the same property noted multiple times. Can you double-check this?

Reply:
Thanks for pointing this out, you can represent stringSet as a comma-delimited string. I've updated this.
http://checkstyle.sourceforge.net/property_types.html#stringSet